client/local/local.go

```diff
@@ -414,6 +414,24 @@ func (lc *Client) TailDaemonLogs(ctx context.Context) (io.Reader, error) {
 	return res.Body, nil
 }
 
+// TailBusEvents returns a stream the Tailscale bus events as they arrive.
```

Comment:
This is fine as-written, but I was confused at first not realize you meant "tail" in the "watch the end of the stream" sense rather than as a "Tailscale" modifier.

Perhaps `StreamBusEvents`?

I think we should also mention the expected output format (line-delimited JSON probably?). Should we also ensure the reader gets a consistent error after ctx ends, like `io.EOF`? (Assuming we have any control over it)

Reply:
Yeah, I think returning an `io.ReadCloser` as you have done addresses my concern, as it ensures we have a way for the reader to say it's going away.